### ISSUE TITLE
Users won't be imported into MAS following an update from an existing Maximo Manage database

### ISSUE THREAD

**Issue by jroldj** [ibm-mas/ansible-devops#532]
I've discovered that after doing a Maximo Manage database update or upgrade, users that were established from an existing MAS environment won't be imported to a new MAS environment.

For instance, if I use a SQL database to deploy Maximo Manage 8.4 with demo data in MAS 8.8, and then upgrade to Maximo Manage 8.5 and MAS 8.9 later, the existing out-of-the-box users, including those that were manually created in MAS 8.8, will not be imported to the MAS once the upgrade or update process is completed.

How can these users be imported during the upgrade process be forced?

I am aware that users generated in MAS are kept in a Mongo database, though.

Thanks,
Jean

**Comment by whitfiea** [comment]
@jroldj I don't believe this should be the case. An upgrade shouldn't cause users to be lost. Just to clarify, when you say `users that were established from an existing MAS environment won't be imported to a new MAS environment` this implies you are moving to a new instance rather than upgrading the same instance. Is this an upgrade via OLM or via the install script? What is the upgrade process you are following? After the upgrade you was left with no users other than the superuser?

**Comment by jjiraud** [comment]
@whitfiea Thanks for your comments.

Let me just give you an idea of the upgrade steps I followed:

1) Upgrade Classic Maximo database 7.6.1.2 to MAS 8.8 and Manage 8.4
   Result: All exisiting Classic Maximo 7.6.1.2 users will be automatically imported into MAS 8.8 and Manage 8.4

2) Backup Maximo Manage database 8.4 which also contains the exisisting users from classic Maximo 7.6.1.2


3) Install MAS 8.9

4) Update JDBC configuration(in MAS GUI) to point to Manage 8.4 DB and initiate the activation process to update the Database.
   Result: Maximo Manage 8.5 will be updated and ready to use, howerver the users will not be imported to MAS 8.9 automatically like Maximo 7.6.1.2 upgrade to MAS.

**Additional note:** 
I am aware that this method may not be regarded as the best way to upgrade a Maximo Manage database from one version to another, but taking into account a scenario where MAS and Openshift failed 
to recover and the only thing left is the Manage database that runs outside of the OpenShift cluster, this would be the best way to rebuild the environment if a similar incident ever occurred.
Additionally, I understand that those users may be effectively recreated in the new MAS 8.9 and log in to Manage successfully. 

Thanks,
Jean

**Comment by jjiraud** [comment]
You might be right!

**Comment by whitfiea** [comment]
@andrercm It needs input from someone from the Manage team to confirm that the steps outlined in https://github.com/ibm-mas/ansible-devops/issues/532#issuecomment-1416255981 are what would happen as I am not sure what the upgrade in step 1 is doing as this is something provided by the Manage team.

**Comment by wangqymro** [comment]
This is not a bug. As the database has already been upgraded to 8.x, the users are already synced and marked so in the database. Even it is new install of MAS, the sync is not happening again. This is not a valid scenario. If it is desired, there is way in the manage db to be able to trigger a resync again.

**Comment by jjiraud** [comment]
@wangqymro, thanks for your input.  Can you please share any technical procedure to trigger the resync against the Manage database? 

Thanks,
Jean
